proxy/tests/support/proxy.rs

```diff
@@ -111,6 +111,10 @@ fn run(proxy: Proxy) -> Listening {
     env.put(config::ENV_PUBLIC_LISTENER, "tcp://127.0.0.1:0".to_owned());
     env.put(config::ENV_CONTROL_LISTENER, "tcp://127.0.0.1:0".to_owned());
 
+    env.put(config::ENV_POD_NAMESPACE, "conduit".to_owned());
+    env.put(config::ENV_POD_ZONE, "local".to_owned());
```

Comment:
Did you mean for this to be "cluster.local", which is what we normally expect things to be? If not, then it would be useful to add a comment about why you chose "local" specifically. If you just want an arbitrary value then use "example" instead.

Reply:
I actually just chose those values since all the existing tests in `proxy/tests` use the authority `test.conduit.local`.

Reply:
OK, those tests are using "local" to make sure we don't match just "local" instead of "cluster.local". If they're using "conduit" then that's a mistake on my part.